neqo-transport/src/connection/tests/stream.rs

```diff
@@ -111,6 +113,92 @@ fn transfer() {
     assert!(fin3);
 }
 
+#[derive(PartialEq, Eq, PartialOrd, Ord)]
+struct IdEntry {
+    sendorder: StreamOrder,
+    stream_id: StreamId,
+}
+
+#[cfg(test)]
+// tests stream sendorder priorization
+fn sendorder_test(order_of_sendorder: &[Option<SendOrder>]) {
```

Comment:
Can we have a similar test in `neqo-http3/src/features/extended_connect/tests/webtransport/streams.rs`?

Reply:
I think that transport tests will suffice.  Though a much simpler test there might test that the plumbing works.  Say make a webtransport, write to two streams, set sendorder, check that sendorder overrides stream ID order.